### PR TITLE
Init Project

### PR DESCRIPTION
**Pull request by artoursong**

### CODE REVIEW

**Comment by artoursong** [comment]
ready

**Comment by artoursong** [comment]
ready

**Comment by artoursong** [comment]
ready

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find(params[:id])
```
Comment (vanvtt-0952) [inline app/controllers/users_controller.rb]: dùng `find_by` thay vì find nhé

nhớ them code xử lý nil

**Comment by artoursong** [comment]
ready

**Comment by vanvtt-0952** [comment]
merged

**Pull request merged**